src/DotNetty.Codecs/Base64/Base64Encoder.cs

```diff
@@ -0,0 +1,32 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+namespace DotNetty.Codecs.Base64
+{
+    using System;
+    using System.Collections.Generic;
+    using DotNetty.Buffers;
+    using DotNetty.Transport.Channels;
+
+    public class Base64Encoder : MessageToMessageDecoder<IByteBuffer>
```

Comment:
>Base64Encoder [](start = 17, length = 13)

mark as sealed. Also please override `Shareable` and return `true` as the encoder itself has no transient state and can be shared across channels